crypto_kem/mceliece348864/META.yml

```diff
@@ -34,7 +34,7 @@ implementations:
             - Linux
             - Darwin
           required_flags:
-            - sse4_1
+            - sse4.1
```

Comment:
Ah, this is actually necessary due to it being reported as `sse4_1` by `/proc/cpuinfo`...